logbook-spring-boot3-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java

```diff
@@ -0,0 +1,409 @@
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.core.Ordered;
+import org.springframework.security.web.SecurityFilterChain;
+import org.zalando.logbook.BodyFilter;
+import org.zalando.logbook.BodyOnlyIfStatusAtLeastStrategy;
+import org.zalando.logbook.ChunkingSink;
+import org.zalando.logbook.Conditions;
+import org.zalando.logbook.CorrelationId;
+import org.zalando.logbook.CurlHttpLogFormatter;
+import org.zalando.logbook.DefaultCorrelationId;
+import org.zalando.logbook.DefaultHttpLogFormatter;
+import org.zalando.logbook.DefaultHttpLogWriter;
+import org.zalando.logbook.DefaultSink;
+import org.zalando.logbook.DefaultStrategy;
+import org.zalando.logbook.HeaderFilter;
+import org.zalando.logbook.HeaderFilters;
+import org.zalando.logbook.HttpLogFormatter;
+import org.zalando.logbook.HttpLogWriter;
+import org.zalando.logbook.HttpRequest;
+import org.zalando.logbook.Logbook;
+import org.zalando.logbook.PathFilter;
+import org.zalando.logbook.PathFilters;
+import org.zalando.logbook.QueryFilter;
+import org.zalando.logbook.QueryFilters;
+import org.zalando.logbook.RequestFilter;
+import org.zalando.logbook.RequestFilters;
+import org.zalando.logbook.ResponseFilter;
+import org.zalando.logbook.ResponseFilters;
+import org.zalando.logbook.Sink;
+import org.zalando.logbook.SplunkHttpLogFormatter;
+import org.zalando.logbook.StatusAtLeastStrategy;
+import org.zalando.logbook.Strategy;
+import org.zalando.logbook.WithoutBodyStrategy;
+import org.zalando.logbook.httpclient.LogbookHttpRequestInterceptor;
+import org.zalando.logbook.httpclient.LogbookHttpResponseInterceptor;
+import org.zalando.logbook.json.JsonHttpLogFormatter;
+import org.zalando.logbook.servlet.LogbookFilter;
+import org.zalando.logbook.servlet.SecureLogbookFilter;
+import org.zalando.logbook.spring.LogbookClientHttpRequestInterceptor;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.function.Predicate;
+
+import static jakarta.servlet.DispatcherType.ASYNC;
+import static jakarta.servlet.DispatcherType.REQUEST;
+import static org.apiguardian.api.API.Status.INTERNAL;
+import static org.apiguardian.api.API.Status.STABLE;
+import static org.zalando.logbook.BodyFilters.defaultValue;
+import static org.zalando.logbook.BodyFilters.truncate;
+import static org.zalando.logbook.HeaderFilters.replaceHeaders;
+import static org.zalando.logbook.QueryFilters.replaceQuery;
+import static org.zalando.logbook.autoconfigure.LogbookAutoConfiguration.ServletFilterConfiguration.newFilter;
+
+@API(status = STABLE)
+@Configuration(proxyBeanMethods = false)
```

Comment:
This and `@AutoConfigureAfter` should be replaced with `@AutoConfiguration(after = { JacksonAutoConfiguration.class, WebMvcAutoConfiguration.class } )`

Reply:
done